modules/aerodyn/src/AeroAcoustics_Types.f90

```diff
@@ -73,7 +73,7 @@ MODULE AeroAcoustics_Types
 ! =======================
 ! =========  AA_InputFile  =======
   TYPE, PUBLIC :: AA_InputFile
-    REAL(DbKi)  :: DT_AA      !< Time interval for aerodynamic calculations {or "default"} [s]
+    REAL(DbKi)  :: DT_AA      !< Time interval for aerodynamic calculations {or  [default"}"]
```

Comment:
The comments/units here are a little bit off... probably due to the way quotation marks are used in the registry input file.